assets/index.css

```diff
@@ -0,0 +1,1107 @@
+h2 {
+  line-height: 0.8;
+}
+
+body {
+  margin: 0 auto;
+  max-width: 1310px;
+  color: #333;
+  font-size: 13px;
+  line-height: 18px;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #f4f5f7;
+}
+
+svg {
+  fill: white;
+  width: 8rem;
+  cursor: pointer;
+}
+
+header {
+  background-color: #f72210;
+  font-family: 'Roboto Mono', monospace;
+}
+
+.container {
+  max-width: 1310px;
+}
+
+.header-content {
+  padding: 0.7rem 0;
+}
+
+.header-subscribe {
+  display: flex;
+}
+
+.header-user-icon {
+  color: rgba(255, 255, 255, 0.75);
+  font-size: 1rem;
+  padding-right: 0.62rem;
+}
+
+.subscribe-button {
+  font-weight: bolder;
+  background-color: #ffe400;
+  color: black;
+  border: none;
+  border-radius: 0.18rem;
+  padding: 0 0.3rem;
+  height: 1.5rem;
+  font-size: 1rem;
+}
+
+.header-menu {
+  font-size: 1rem;
+  padding: 0 0.8rem;
+  padding-top: 0.29rem;
+  color: rgba(255, 255, 255, 0.55);
+}
+
+.header-login {
+  display: none;
+  color: white;
+  font-weight: bolder;
+  padding-right: 1rem;
+  font-size: 1.1rem;
+}
+
+.date-info {
+  display: none;
+}
```

Comment:
You can use the `d-none` bootstrap class here. Please see this [cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) for more information